src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/enums/ToDoType.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.adjustments.api.enums
+
+enum class ToDoType {
+  ADA_INTERCEPT,
```

Comment:
I think the "Thing to do" for ADAs wants to know what kind of intercept it is. Not sure if that makes it a separate type of "Thing to do" or not!

Reply:
See for CMG-169 for the different scenarios we need to  differentiate between